### DISCUSSION TITLE
Eevee next

### DISCUSSION THREAD

**Discussion by e2002e** [UPBGE/upbge#1751] [Announcements & others]
Hi, not an issue just a question.
Got eevee next in alpha 3.4 from the website. But no settings, only shaders I can add is a principled volume.
I saw something about eevee being function by the end of the week, but cannot find the article again.
Is it usable ?

**Comment by youle31** [comment]
eevee-next is a WIP. Still crashing on my other PC, only material preview, idk even know if shadows are there.
